config/src/main/java/com/megaease/easeagent/config/ConfigFactory.java

```diff
@@ -17,54 +17,67 @@
 
 package com.megaease.easeagent.config;
 
+import com.google.common.base.CaseFormat;
+import com.google.common.base.Strings;
 import com.megaease.easeagent.log4j2.Logger;
 import com.megaease.easeagent.log4j2.LoggerFactory;
+import com.megaease.easeagent.plugin.utils.ImmutableMap;
 import com.megaease.easeagent.plugin.utils.SystemEnv;
 import com.megaease.easeagent.plugin.utils.common.JsonUtil;
 import com.megaease.easeagent.plugin.utils.common.StringUtils;
+import io.opentelemetry.sdk.resources.OtelSdkConfigs;
 
 import java.io.File;
-import java.util.*;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
 
 public class ConfigFactory {
     private static final Logger LOGGER = LoggerFactory.getLogger(ConfigFactory.class);
     private static final String CONFIG_PROP_FILE = "agent.properties";
     private static final String CONFIG_YAML_FILE = "agent.yaml";
 
-    private static final String AGENT_SERVICE_NAME = "easeagent.name";
-    private static final String AGENT_SYSTEM_NAME = "easeagent.system";
+    public static final String AGENT_CONFIG_PATH = "config.path";
 
-    private static final String AGENT_SERVER_PORT_KEY = "easeagent.server.port";
-    private static final String AGENT_SERVER_ENABLED_KEY = "easeagent.server.enabled";
+    public static final String AGENT_SERVICE = "name";
+    public static final String AGENT_SYSTEM = "system";
+
+    public static final String AGENT_SERVER_PORT = "easeagent.server.port";
+    public static final String AGENT_SERVER_ENABLED = "easeagent.server.enabled";
 
     public static final String EASEAGENT_ENV_CONFIG = "EASEAGENT_ENV_CONFIG";
 
-    private static final List<String> subEnvKeys = new LinkedList<>();
-    private static final List<String> envKeys = new LinkedList<>();
+    private static final Map<String, String> AGENT_CONFIG_KEYS_TO_PROPS =
+        ImmutableMap.<String, String>builder()
+            .put("easeagent.config.path", AGENT_CONFIG_PATH)
```

Comment:
```suggestion
```
Don't put "easeagent.config.path" in "config", we need concise code: get "config" from "easeagent.config.path", and then get the easeagent configuration from "config". Rather than the other way around you need to get "easeagent.config.path" from "config"